src/Symfony/Component/AssetMapper/Compiler/AssetCompilerPathResolverTrait.php

```diff
@@ -47,4 +56,35 @@ private function resolvePath(string $directory, string $filename): string
 
         return implode('/', $output);
     }
+
+    private function createRelativePath(string $fromPath, string $toPath): string
+    {
+        $fromPath = rtrim($fromPath, '/');
+        $toPath = rtrim($toPath, '/');
+
+        $fromParts = explode('/', $fromPath);
```

Comment:
Quick question: are directory separators normalized before calling this function? Should we use `DIRECTORY_SEPARATOR` constant? Does PHP handle all this automatically so this is not important?

Reply:
Good question. I've been relying on a strong test suite and CI to catch Windows problems. For the most part, paths inside of AssetMapper use `/` always and we normalize for that. In this case, the paths passed to this are the "public paths", which will be the "asset prefix" (e.g. `/assets/`, which is configurable, but it's meant to be the URL so `/` should be used) + the "logical path name" of the asset, which is also normalized to `/`). We may have some blind spot here for Windows... but I'm not aware of it yet. And we can fix those when they come up.